lib/galaxy/model/__init__.py

```diff
@@ -185,7 +186,7 @@ def get_disk_usage( self, nice_size=False ):
         if self.disk_usage is not None:
             rval = self.disk_usage
         if nice_size:
-            rval = galaxy.datatypes.data.nice_size( rval )
+            rval = galaxy.util.nice_size( rval )
```

Comment:
Yes, `nice_size` is also a parameter of this function (and of all the others below except one).